lib/HTML/Nodes/DefinitionListNode.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\HTML\Nodes;
+
+use Doctrine\RST\Nodes\DefinitionListNode as Base;
+use function count;
+use function sprintf;
+
+class DefinitionListNode extends Base
+{
+    public function render() : string
```

Comment:
Can this method be split up a bit? I see at least 3 levels of nesting. Naming the sections and moving them to `private` methods would help a bit